magnet_error_pid:
  kp: 3.0
  ki: 0.0
  kd: 0.0
pure_pursuit:
  mode: fixed  # 1) [fixed] lookahead distance 2) [variable] k_pre * v
  lookahead_distance: 4.0 # [m]
  k_pre: 1.5
control_switch:
  longitudinal: true # Start: true; Shutdown: false
  lateral: true
longitudinal_mode: 1 # 1: constant speed 2: planned speed 3: desired distance
desired_speed: 1
desired_distance: 5 # m
max_steer_angle: 40.0 
max_magnetic_missing_time: 500 # /25s
lane_mid_reference: 260
visual_scale: 35
enable_visual_control: false
always_enable_manual_switch: false
always_enable_remote_control: false
obstacle_dist_threshold: 2.5
turn_right_rfid_tag: 185